src/pages/planters/[planterid].js

```diff
@@ -242,7 +242,10 @@ export default function Planter(props) {
                 />
               </Box>
               <Box sx={{ mt: 2 }}>
-                <Info iconURI={LocationIcon} info="Shirimatunda, Tanzania" />
+                <Info
+                  iconURI={LocationIcon}
+                  info={`${planter.country_name}, ${planter.continent_name}`}
```

Comment:
@dadiorchen can we assume the planter always has these fields or do we need to add some checks?

Reply:
what is the fallback value that can be added?

Reply:
The fallback should probably be "Unknown"

Reply:
should i add fallback for both separately?

> info={`${planter.country_name ?? 'Unknow'}, ${planter.continent_name ?? 'Unknown'}`}